src/main/webapp/js/common.es6

```diff
@@ -538,24 +538,61 @@ function toggleSort(divElement, comparator) {
 
     const $selectedDivElement = $(divElement);
 
-    if ($selectedDivElement.attr('class') === 'button-sort-none') {
+    if ($selectedDivElement.attr('class') === 'button-sort-none toggle-sort') {
         sortTable(divElement, colIdx, comparator, true, row);
-        $selectedDivElement.parent().find('.button-sort-ascending').attr('class', 'button-sort-none');
-        $selectedDivElement.parent().find('.button-sort-descending').attr('class', 'button-sort-none');
+        $selectedDivElement.parent().find('.button-sort-ascending').attr('class', 'button-sort-none toggle-sort');
+        $selectedDivElement.parent().find('.button-sort-descending').attr('class', 'button-sort-none toggle-sort');
```

Comment:
If the source of problem is the class name got entirely overwritten, then the solution is to overwrite only what's necessary, right?
```js
.removeClass('button-sort-descending').addClass('button-sort-none');
```